CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - COBOL language configuration for highlighting matching brackets and auto-insertion of line numbers in fixed-format code [#330](https://github.com/OCamlPro/superbol-studio-oss/pull/330)
 
 ### Fixed
+- Fix completion on some parser state [#322](https://github.com/OCamlPro/superbol-studio-oss/pull/322)
```

Comment:
```suggestion
- Completion on some parser states [#322](https://github.com/OCamlPro/superbol-studio-oss/pull/322)
```

Reply:
Actually that'd be in `Added` Subsection, and substituting "for more grammar constructs" for "on some parser states".